cfgov/unprocessed/css/apps/owning-a-home/form-explainer.less

```diff
@@ -0,0 +1,447 @@
+/* ==========================================================================
+   "Form explainer" custom styles
+   ========================================================================== */
+
+.form-explainer {
+    .u-webfont-regular();
+
+    .u-right-on-desktop {
+        .respond-to-min( 801px, {
+            text-align: right;
+        } );
+    }
+
+    .page-intro {
+        padding-bottom: 36px;
+    }
+
+    .illustration {
+        padding-top: 1em;
+        width: 200px;
+        margin: 0 auto;
+        display: block;
+        .respond-to-max( 719px, {
+            display: none;
+        } );
+    }
+
+    .rateq {
+        margin-top: 0px;
```

Comment:
Any idea why this is needed? If it is, could you remove the `px`, 0 doesn't need a unit type.

Reply:
This was removed, it was needed.